apps/prairielearn/src/server.ts

```diff
@@ -492,6 +492,13 @@ export async function initExpress(): Promise<Express> {
         .default,
     ],
   );
+  app.use(
+    '/pl/assessments_switcher/course_instance/:course_instance_id(\\d+)/assessment/:assessment_id(\\d+)?',
```

Comment:
Nit: can we use the same `/pl/navbar` namespace as the above routes? I'd propose this (you'll have to update the URL elsewhere):

```suggestion
    '/pl/navbar/course_instance/:course_instance_id(\\d+)/assessment/:assessment_id(\\d+)/switcher',
```

Aside: can you say more about why the assessment ID bit of the route was listed as optional? AFAIK we should always be rendering this menu in the context of a specific assessment. If it doesn't have to be optional, let's fix that and update the relevant types on this page.

Reply:
> `/pl/navbar` namespace
Sounds good; I'll make this change 👍 

> Assessment ID listed as optional
The assessment ID was listed as optional since I initially assumed that it could be called if no assessment ID was selected. If no assessment ID was specified, the HTML for the assessment switcher would be returned, just without an assessment selected. 

Since this menu is always rendered in the context of a specific assessment, I'll set the assessment ID to be required.

Reply:
Makes sense, thanks!